scripts/check-ancient-pip.sh

```diff
@@ -7,19 +7,19 @@ PYTHON=$1
 
 BROKEN_VIRTUALENV=$($PYTHON -c'import tempfile; print(tempfile.mkdtemp())')
 
-trap "rm -rf $BROKEN_VIRTUALENV" EXIT
+trap 'rm -rf $BROKEN_VIRTUALENV' EXIT
```

Comment:
It's not totally obvious to me that this is doing the right thing, because I don't understand the semantics of variable evaluation in trap. Previously this would have evaluated the BROKEN_VIRTUALENV variable at point of definition. Does it evaluate it at point of execution now and is that variable in scope there?